packages/hardhat-core/src/internal/cli/project-creation.ts

```diff
@@ -239,26 +242,37 @@ export async function createProject() {
     shouldAddGitAttributes: boolean;
   };
 
-  try {
-    responses = await enquirer.prompt<typeof responses>([
-      {
-        name: "projectRoot",
-        type: "input",
-        initial: process.cwd(),
-        message: "Hardhat project root:",
-      },
-      createConfirmationPrompt(
-        "shouldAddGitIgnore",
-        "Do you want to add a .gitignore?"
-      ),
-    ]);
-  } catch (e) {
-    if (e === "") {
-      return;
-    }
+  const useDefaultPromptResponses =
+    process.env.HARDHAT_CREATE_SAMPLE_PROJECT_WITH_DEFAULTS !== undefined;
+
+  if (useDefaultPromptResponses) {
+    responses = {
+      projectRoot: process.cwd(),
+      shouldAddGitIgnore: true,
+      shouldAddGitAttributes: true,
```

Comment:
We should remove this variable, it's not used anymore now that GitHub has support for Solidity.

Reply:
Addressed in https://github.com/nomiclabs/hardhat/pull/1686/commits/afdbe683bf2f10cda8d770728c94bf56fa9b7c5e